### PR TITLE
Requesting xengsort=2.0.5 and other dependencies

### PR DESCRIPTION
**Pull request by tamuanand**
Requesting xengsort=2.0.5,pigz=2.8,htop=3.2.2,seqkit=2.7.0,csvtk=0.29.0

### CODE REVIEW

**Comment by tamuanand** [comment]
Hi @bgruening - Requesting review/merge. Thanks

File: combinations/hash.tsv [hunk 1]
```diff
@@ -551,3 +551,4 @@ r-hwriter=1.3.2,r-ggplot2=3.4.4,r-ini=0.3.1,r-ashr=2.2_63,r-dplyr=1.1.4,r-ggrepe
 pandas=2.2.1,tensorflow=2.15.0,scikit-learn=1.4.1,umap-learn=0.5.5,matplotlib=3.8.0
 magus-msa=0.1.3,pigz=2.8
 gatk4=4.5.0.0,samtools=1.19.2,bcftools=1.19,perl=5.32.1,r-base=4.3
+xengsort=2.0.5,pigz=2.8,htop=3.2.2,seqkit=2.7.0,csvtk=0.29.0
```
Comment (bgruening) [inline combinations/hash.tsv:554]: htop? This should not be needed by your workflow isn't it?

Comment (tamuanand) [reply combinations/hash.tsv:554]: It is needed for debugging when the job runs. But if that is a show-stopper for this, by all means we can remove that.'

Do you want me to update

**Comment by tamuanand** [comment]
hi @bgruening - I have removed htop and this is ready for review/merge

**Comment by tamuanand** [comment]
Hi @bgruening 
I was wondering if there are any issues with merging this

Question: Is there a page 

- which lists all possible reviewers who can review/merge these requests? 
- what turrnaround times should one expect with these merge once all tests pass

Thanks

**Comment by bgruening** [comment]
All biocontainers team members can merge here afaik. Turnaround time is around 1-2 working days.

**Comment by tamuanand** [comment]
Ok - thanks a lot for the info. Should the biocontainers team members be pinged when there is something ready for review/merge?

Suggestion: It might be best to capture that info on the main page, 
```
All biocontainers team members can merge here afaik. Turnaround time is around 1-2 working days.
```

**Comment by bgruening** [comment]
As with all organisations on GitHub, you find the members at https://github.com/orgs/BioContainers/people

I'm not sure if pinging helps, usually, we all get notifications. A bit more patience especially over weekends might be good :)

**Comment by tamuanand** [comment]
Is it worth considering Github Actions to auto-merge after successful tests - avoiding manual intervention. Something to consider? 

https://stackoverflow.com/a/71635816/10101904

**Pull request merged**